projects/1.16/assets/geolosys/geolosys/lang/zh_cn.json

```
@@ -1 +1,103 @@
-{}
\ No newline at end of file
+{
+  "block.geolosys.rhododendron": "杜鹃",
+  "block.geolosys.peat": "泥炭",
+  "block.geolosys.lignite_ore": "褐煤",
+  "block.geolosys.bituminous_coal_ore": "烟煤",
+  "block.geolosys.anthracite_coal_ore": "褐煤",
+  "block.geolosys.hematite_ore": "赤铁矿",
+  "block.geolosys.limonite_ore": "褐铁矿",
+  "block.geolosys.malachite_ore": "孔雀石",
+  "block.geolosys.azurite_ore": "蓝铜矿",
+  "block.geolosys.cassiterite_ore": "锡石",
+  "block.geolosys.teallite_ore": "硫锡铅矿",
+  "block.geolosys.galena_ore": "方铅矿",
+  "block.geolosys.bauxite_ore": "铝土矿",
+  "block.geolosys.autunite_ore": "钙铀云母",
+  "block.geolosys.platinum_ore": "铂",
+  "block.geolosys.sphalerite_ore": "闪锌矿",
+  "block.geolosys.coal_ore": "煤",
+  "block.geolosys.cinnabar_ore": "朱砂",
+  "block.geolosys.gold_ore": "金",
+  "block.geolosys.lapis_ore": "青金石",
+  "block.geolosys.quartz_ore": "杂石英",
+  "block.geolosys.kimberlite_ore": "金伯利岩",
+  "block.geolosys.beryl_ore": "绿柱石",
+  "block.geolosys.ancient_debris_ore": "古代遗迹",
```

|Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

```suggestion
  "block.geolosys.ancient_debris_ore": "远古残骸",
```